src/components/navbar/NavbarComponent.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import NavbarLogoComponent from './logo/NavbarLogoComponent.tsx';
+import NavbarMenuComponent from './navmenu/NavbarMenuComponent.tsx';
+
+const NavbarComponent = () => {
+	return (
+		<nav className='navbar__container'>
+			<NavbarLogoComponent />
```

Comment:
Jakby svg wyrzucić do pliku to możnaby użyć tutaj SVG, chyba że tak specjalnie zrobiłeś w ramach czytelności?